Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM python:3.11-alpine
-COPY src/app src
-COPY requirements.txt requirements.txt
+COPY requirements.txt .
 RUN pip install -r requirements.txt
+COPY src/app src
 ENV PYTHONPATH=src
-CMD exec uvicorn src.app:app --host 0.0.0.0 --port $PORT
+CMD ["uvicorn", "src.app:app", "--host", "0.0.0.0", "--port", "8080"]
```

Comment:
`CMD ["sh", "-c", "uvicorn", "src.app:app", "--host", "0.0.0.0", "--port", "$PORT"]`
Can we execute the shell in the exec form as mentioned here https://docs.docker.com/reference/dockerfile/#shell-and-exec-form ? Might need to check if Fastapi like it.

Reply:
as per docs kept uvicorn command in 1 element which worked :)